docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+services:
+  web:
+    image: dockerimage:latest
```

Comment:
We could use build instead of image which would remove the need to manually build in advance.

Reply:
Regarding this, I found that doing `build: .` instead of the image wasn't working as intended. Was very confusing, so using image was the only way I found to make sure it always worked. Effectively using `build: .` just wasn't updating the files (I exec'd into the docker container to check this was the case). Any clues on what I may have been doing wrong?